petclinic.yaml

```diff
@@ -11,16 +11,21 @@ tools:
     type: dockerImage
     image: maven:3.5.4-jdk-8
     entryPoint: sleep infinity
-    persistentVolumes:
+    volumes:
       - name: maven-repo
-        containerPath: /root/.m2
-    services:
+        mountPath: /root/.m2
+      - name: projects
+        mountPath: /projects
+    endpoints:
       - name: spring-boot
-        port: 8080
-    mountProjectsSources: true
+        public: true
```

Comment:
We should be careful adding too many fields here. The main goal is to make it simple to write a Devfile. For instance fields `public` and `attributes` should be optional.

Reply:
question, why public is not an attribute (for example secure is an attribute) ?